pkg/component/shared/istio.go

```diff
@@ -80,6 +81,10 @@ func NewIstio(
 		// replicas, which apparently works in all known Gardener scenarios. Reducing it to less
 		// per zone gives some room for autoscaling while it is assumed to never reach the maximum.
 		maxReplicas = ptr.To(len(zones) * 6)
+		if features.DefaultFeatureGate.Enabled(features.IstioTLSTermination) {
+			// When IstioTLSTermination server is enabled, more resources might be required on seeds.
+			maxReplicas = ptr.To(len(zones) * 8)
+		}
```

Comment:
Is there a particular reason why you only increased the amount of replicas for the default istio ingress gateway and not also for the zone-specific ones?
It looks like before this change `maxReplicas` was `18` for a seed with three availability zones. This applies only to the default istio ingress gateway. The zone-specific istio ingress gateways had `9` as `maxReplicas` (see [here](https://github.com/gardener/gardener/blob/aca92c553522ae4351b91bc093b19fda5bd707cf/pkg/component/networking/istio/charts/istio/istio-ingress/values.yaml#L29)).
After the change the values are `24` and `9` respectively.